public/content/docs/mels-hack/index.en.md

```diff
@@ -0,0 +1,173 @@
+---
+title: "Mel's hack - the missing bits"
+author: "(David *)Frenkiel"
+date: Thu Jun 23 2022 01:14:31 GMT+0300
+---
+
+### A private _Mel moment_
+
+        When the light went on it nearly blinded me. 
```

Comment:
Should be carried out as Blockquote, as well as other textual references to the codex.

Reply:
Let's postpone this fix until I implement blockquote merging (merge consecutive blockquotes into one element)